src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -128,11 +208,18 @@
   margin-left: auto;
 }
 
+.pf-c-notification-drawer__tabs {
+  background-color: var(--pf-c-notification-drawer__header--BackgroundColor);
```

Comment:
Since this isn't part of the header, can this be it's own var? Also if we were to ever separate the background color vars for these two in the future, the way it is here would technically be a breaking change.